web/src/components/embedding/LiteLLMModelForm.tsx

```diff
@@ -0,0 +1,132 @@
+import { CloudEmbeddingModel } from "./interfaces";
+import { Formik, Form } from "formik";
+import * as Yup from "yup";
+import { TextFormField, BooleanFormField } from "../admin/connectors/Field";
+import { Dispatch, SetStateAction } from "react";
+import { Button } from "@tremor/react";
+
+export function LiteLLMModelForm({
+  setShowTentativeModel,
+  currentValues,
+}: {
+  setShowTentativeModel: Dispatch<SetStateAction<CloudEmbeddingModel | null>>;
+  currentValues: CloudEmbeddingModel | null;
+}) {
+  return (
+    <div>
+      <Formik
+        initialValues={
+          currentValues || {
+            model_name: "",
+            model_dim: 1536,
```

Comment:
why this number? I don't think this is a particularly common embedding dim right?

We can maybe do 768